### DISCUSSION TITLE
Detectado problema en cierre de contenedores

### DISCUSSION THREAD

**Discussion by labra** [Arquisoft/faq#54] [Announcements]
Hemos detectado que el workflow [release.yml](https://github.com/Arquisoft/wiq_0/blob/master/.github/workflows/release.yml) del contenedor que estábamos utilizando como plantilla tenía una línea incorrecta (línea 127) para el apagado de los contenedores. 

La línea correcta puede verse [aquí](https://github.com/Arquisoft/wiq_0/blob/f4f8efbb036eb8f73b7d5b0bbc7339e9c48d8d49/.github/workflows/release.yml#L127) y debe contener:

```
docker compose --profile prod down
``` 

Para que funcione correctamente la actualización de contenedores en las releases, todos los equipos deberían actualizar dicha línea.

**Comment by dariogmori** [comment]
Good morning, after many pain and suffering when deploying I have come to the conclusion that there is another line missing in the deploy action. The docker compose up line shall be substituted for:
`docker compose --profile prod up -d --pull always`
This ensures the docker images are always updated when running docker compose :)
Please react with a 👍 emoji if this helped you :)

**Comment by pglez82** [comment]
Yes you are right,

Last year we had a line before the `docker-compose up` that did `docker-compose pull`, but it got removed by mistake.

Anyway, i think both solutions are equivalent but the `--pull always` flag seems a cleaner approach. Thanks!
